src/components/dev-hub/page-helmet.js

```diff
@@ -0,0 +1,43 @@
+import React, { memo } from 'react';
+import PropTypes from 'prop-types';
+
+import SEO from '~components/dev-hub/SEO';
+import { useSiteMetadata } from '~hooks/use-site-metadata';
+
+const STANDARD_DESCRIPTION =
+    "Code, content, tutorials, programs and community to enable developers of all skill levels on the MongoDB Data Platform which includes Atlas, Realm, Compass, Data Lake and more. Whether you're coding in Java, JavaScript, C#, Python, Node, Go or looking for how this fits with IOT, AI, ML - join or follow us here.";
+
+const PageHelmet = ({ canonicalUrl, metaDescription, metaTitle }) => {
```

Comment:
I would also like for us to optionally accept other SEO props as props here. The reason is soon we will be exposing these pages for changes via the CMS and we would like for the team to be able to adjust these if need be.